main.go

```diff
@@ -165,10 +214,200 @@ func convertToV2(v1 models.CircleYamlV1) (models.CircleYamlV2, error) {
 				continue
 			}
 
-			step := map[string]string{"deploy": `if [ "${CIRCLE_BRANCH}" == "master" ]; then ` + item + `; fi;`}
+			step := map[string]string{"run": `if [ "${CIRCLE_BRANCH}" == "master" ]; then ` + item + `; fi;`}
 			v2.Jobs.Build.Steps = append(v2.Jobs.Build.Steps, step)
 		}
 	}
+	return nil
+}
 
-	return v2, nil
+func addCreateCIArtifactDirsStep(v2 *models.CircleYamlV2) {
+	createCIArtifactsDirsStep := map[string]interface{}{
+		"run": map[string]string{
+			"name":    "Set up CircleCI artifacts directories",
+			"command": `mkdir -p $CIRCLE_ARTIFACTS $CIRCLE_TEST_REPORTS`,
+		},
+	}
+	v2.Jobs.Build.Steps = append(v2.Jobs.Build.Steps, createCIArtifactsDirsStep)
+}
+
+func addSetupNPMRCStep(v2 *models.CircleYamlV2) {
+	setupNPMRCStep := map[string]interface{}{
+		"run": map[string]string{
+			"name": "Set up .npmrc",
+			"command": `
+sed -i.bak s/\${npm_auth_token}/$NPM_TOKEN/ .npmrc_docker
+mv .npmrc_docker .npmrc`,
+		},
+	}
+	v2.Jobs.Build.Steps = append(v2.Jobs.Build.Steps, setupNPMRCStep)
+}
+
+func addNPMInstallStep(v2 *models.CircleYamlV2) {
+	npmInstallStep := map[string]interface{}{
+		"run": map[string]string{
+			"name":    "npm install",
+			"command": "npm install",
+		},
+	}
+	v2.Jobs.Build.Steps = append(v2.Jobs.Build.Steps, npmInstallStep)
+}
+
+func addInstallAWSCLIStep(v2 *models.CircleYamlV2) {
+	installAWSCLIStep := map[string]interface{}{
+		"run": map[string]string{
+			"name": "Install awscli for ECR publish",
+			"command": `cd /tmp/ && wget https://bootstrap.pypa.io/get-pip.py && sudo python get-pip.py
+sudo apt-get install python-dev
+sudo pip install --upgrade awscli && aws --version
+pip install --upgrade --user awscli`,
+		},
+	}
+	v2.Jobs.Build.Steps = append(v2.Jobs.Build.Steps, installAWSCLIStep)
+}
+
+func addCloneCIScriptsStep(v2 *models.CircleYamlV2) {
+	cloneCIScriptsStep := map[string]interface{}{
+		"run": map[string]string{
+			"name":    "Clone ci-scripts",
+			"command": `cd $HOME && git clone --depth 1 -v https://github.com/Clever/ci-scripts.git && cd ci-scripts && git show --oneline -s`,
+		},
+	}
+	v2.Jobs.Build.Steps = append(v2.Jobs.Build.Steps, cloneCIScriptsStep)
+}
+
+func determineWorkingDirectory(appType string) (string, error) {
+	// @TODO: determine decent working directory depending on app type for non-(go, wag, node) apps
+	// go, wag: /go/src/github.com/Clever/catapult
+	// non-wag node: ~/Clever/hubble
+
+	org := "Clever"
```

Comment:
I kinda feel like we could just inline this 🤷‍♂️

Reply:
✅ fd14e8957f2188870ddf67acfd916bbb21af9f1e

Reply:
✅ fd14e8957f2188870ddf67acfd916bbb21af9f1e